docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -533,22 +491,22 @@
       ]
     },
     {
+      "cell_type": "markdown",
       "metadata": {
         "id": "fh3p0DtcdOK1",
         "colab_type": "text"
       },
-      "cell_type": "markdown",
       "source": [
         "To further explore the capabilities of the HParams dashboard, download a set of pregenerated logs with more experiments:"
       ]
     },
     {
+      "cell_type": "code",
       "metadata": {
         "id": "oxrSUAnCeFmQ",
         "colab_type": "code",
         "colab": {}
       },
-      "cell_type": "code",
       "source": [
         "%%bash\n",
         "wget -q 'https://storage.googleapis.com/download.tensorflow.org/tensorboard/hparams_demo_logs.zip'\n",
```

Comment:
There is no reason to update this set of logs, right?

Reply:
That was my conclusion. The purpose of those logs is to explore the
dashboard, not to correspond exactly to the experiment code in this
notebook (and they already didn’t). It’s fine if there continue to be
minor deviations.

Reply:
Sounds good.